src/nih_wayfinding/app/scripts/navigation/navigation-service-slopedmock.js

```diff
@@ -0,0 +1,201 @@
+
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function Navigation (
+        $q, $interval, $rootScope
+    ) {
+
+        var stepsLeft = [];
+        var navInterval = 1000; // 1 second default interval
+        var module = {
+            walkTheLine: walkTheLine,
+            getSteps: getSteps,
```

Comment:
Should be private, since this function provides implementation detail and isn't used.